pkg/arvo/mar/s3/action.hoon

```diff
@@ -0,0 +1,8 @@
+/+  *s3-json
+|_  act=action
+++  grab
+  |%
+  ++  noun  action
+  ++  json  |=(jon=^json (json-to-action jon))
```

Comment:
```suggestion
  ++  json  json-to-action
```

Reply:
Done! Love it, will do the same for other apps in the near-term.